scoring/cscore.metta

```diff
@@ -44,6 +44,34 @@
            (() (println! (PenalizedScore: $updatedPScore))))
         
             (mkCscore $scor $cpxy $cpxyPenalty $uniformityPenalty $updatedPScore)))
+;converts a score to an expression
+; (: cScoreToExpr (-> Cscore Expression))
```

Comment:
Is the type causing an issue? Is that why it's commented out? Or was it by accident?